### PR TITLE
Configure Pools page updated  

### PR DESCRIPTION
**Pull request by SanjaySinghRajpoot**
# Pull Request Template

## Description

- Fixed a `:value` vuejs warning 
- Fixed element's position with respect to the website. 

## Additional Context 
Screenshot from 2022-06-25 13-17-01

### CODE REVIEW

File: flint.ui/src/components/GCBM/Configurations/Pools.vue [hunk 1]
```diff
@@ -1,27 +1,39 @@
...
-            step="1"
-            @change="(val) => onPoolChange(poolKey, val)"
-          />
         </div>
         <div>
-          <!-- <code> {{ pool.description }} </code> -->
+            <!-- <code> {{ pool.description }} </code> -->
         </div>
+        <a-input-number
+          v-model="pool.value"
```
Comment (YashKandalkar) [inline flint.ui/src/components/GCBM/Configurations/Pools.vue:29]: What warning were you receiving for this? I'm not getting any warnings- and this doesn't seem to work, the values from the store aren't reflected in the input and the store isn't being updated when changing the inputs. `v-model:value` is [how Ant Design Vue shows it's usage](https://antdv.com/components/input-number).

File: flint.ui/src/components/GCBM/Configurations/Pools.vue [hunk 2]
```diff
@@ -1,27 +1,39 @@
...
+      </span>
     </a-typography-text>
-    <a-input-search v-model:value="searchValue" placeholder="Search for a pool" style="width: 250px" class="mb-6" />
     <a-row :gutter="[16, 16]">
       <a-col v-for="(pool, poolKey) in filteredPools" :key="poolKey" :span="16" :lg="{ span: 12 }" :xl="{ span: 8 }">
-        <div class="flex items-center">
+        <div class="span items-center">
           <a-typography-title :level="5" style="margin: 0">
-            <span class="font-normal mr-4"> {{ poolKey }}: </span>
+            <span class="font-normal mr-4"><b>{{ poolKey }}:</b></span>
```
Comment (YashKandalkar) [inline flint.ui/src/components/GCBM/Configurations/Pools.vue:22]: for making this bold you can just remove the `font-normal` class and remove the `b` tags, as `a-typography-title` is bold by default

File: flint.ui/src/components/GCBM/Configurations/Pools.vue [hunk 3]
```diff
@@ -1,27 +1,39 @@
 <template>
   <div class="py-4 px-8 text-earth">
-    <a-typography-title><span class="font-normal text-earth"> Configure Pools </span></a-typography-title>
+    <a-typography-title
+      ><span class="font-normal text-earth">
+        Configure Pools
+        <a-input-search
```
Comment (YashKandalkar) [inline flint.ui/src/components/GCBM/Configurations/Pools.vue:6]: can you check if the input's position is being rendered correctly? As in the [deployed url](https://gcbm-editor-65lmbieek-moja-global.vercel.app/gcbm/configurations/pools) it's being shown in the middle:
image

**Review by YashKandalkar — CHANGES_REQUESTED** [review]
Hey @SanjaySinghRajpoot, thanks for the changes!
Left a few comments. PTAL.

File: flint.ui/src/components/GCBM/Configurations/Pools.vue (hunk 1 above)
Comment (SanjaySinghRajpoot) [reply flint.ui/src/components/GCBM/Configurations/Pools.vue:29]: https://eslint.vuejs.org/rules/no-v-model-argument.html

Comment (SanjaySinghRajpoot) [reply flint.ui/src/components/GCBM/Configurations/Pools.vue:29]: If this is not working then I will revert it back to the original state.

File: flint.ui/src/components/GCBM/Configurations/Pools.vue (hunk 2 above)
Comment (SanjaySinghRajpoot) [reply flint.ui/src/components/GCBM/Configurations/Pools.vue:22]: ok

**Review by YashKandalkar — COMMENTED** [review]
@SanjaySinghRajpoot  can we make the input position responsive?

File: flint.ui/src/components/GCBM/Configurations/Pools.vue [hunk 4]
```diff
@@ -6,7 +6,7 @@
         <a-input-search
           v-model:value="searchValue"
           placeholder="Search for a pool"
-          style="width: 250px; margin-left: 28rem; margin-top: 2px"
+          style="width: 250px; margin-left: 55rem; margin-top: 2px"
```
[the remark this answers is not in the document]
Comment (SanjaySinghRajpoot) [reply flint.ui/src/components/GCBM/Configurations/Pools.vue]: Done

**Review by YashKandalkar — APPROVED** [review]

**Comment by YashKandalkar** [comment]
LGTM! @SanjaySinghRajpoot Thanks for working on this! :rocket:

**Pull request merged**